File: src/test/java/eotree/TestEOMeta.java
package eotree;

import static org.junit.jupiter.api.Assertions.assertEquals;

import org.junit.jupiter.api.Test;

/**
 * EO meta tests.
 */
public class TestEOMeta {

    @Test
    public void testGenerateEOZeroIndent() {
        var f = new EOMeta("key", "value");
        assertEquals("+key value", f.generateEO(0));
    }

    @Test
    public void testGenerateEONonZeroIndent() {
        var f = new EOMeta("key", "value");
        assertEquals("+key value", f.generateEO(1));
    }
}

File: src/test/kotlin/eotree/TestEOMeta.kt
package eotree

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test
import org.polystat.j2eo.eotree.EOMeta

/**
 * EO meta tests.
 */
class TestEOMeta {
    @Test
    fun testGenerateEOZeroIndent() {
        val f = EOMeta("key", "value")
        Assertions.assertEquals("+key value", f.generateEO(0))
    }

    @Test
    fun testGenerateEONonZeroIndent() {
        val f = EOMeta("key", "value")
        Assertions.assertEquals("+key value", f.generateEO(1))
    }
}
